tasks/main.yml

```diff
@@ -25,10 +25,19 @@
     update_cache: '{{ __systemd_timesyncd_virtualized }}'
   become: yes
 
+- name: Create drop-in configuration folder
```

Comment:
nitpick: for now, let's stick to the lower-cased style of task naming, I really do dislike the general recommendation to require upper-cased task names :grimacing: